current/asvs.yaml

```diff
@@ -1619,15 +1631,17 @@
     nist: 5.1.1.2
     concepts:
         -
-            concept: ''
-            terms: ''
-            level: ''
+            concept: 'Authentication'
+            terms:
+                - 'credential rotation'
+                - 'password history'
+            level: 2
     actions:
         -
-            action: ''
-            level: ''
+            action: 'verify that'
+            level: 2
             mappings:
-                -
+                - 'Authentication'
     notes: ''
```

Comment:
This requirement should specify it applies to user account passwords.

Reply:
See following commit for notes.